warehouse/query-core/src/test/resources/datawave/query/QueryLogicFactory.xml

```diff
@@ -238,6 +238,9 @@
         <property name="eventNextSeek" value="-1" />
         <property name="tfFieldSeek" value="-1" />
         <property name="tfNextSeek" value="-1" />
+        <!--    control when to log warnings for long aggregations    -->
+        <property name="docAggregationThresholdMs" value="-1" />
+        <property name="tfAggregationThresholdMs" value="-1" />
```

Comment:
Do we need to set this since we're just using the default value here?

Reply:
It's not strictly necessary, the example here is for illustration